internal/driver/config.go

```diff
@@ -30,6 +30,10 @@ type config struct {
 	Normalize           bool    `json:"normalize,omitempty"`
 	Sort                string  `json:"sort,omitempty"`
 
+	// Label Psuedo-stack-frame generation options
```

Comment:
"pseudo" too

Also nit: lowercase (not a proper name)

Reply:
thanks, appreciate you pointing it out, i've probably been spelling this wrong all my life :-)